src/other/build_scripts/GolangSimple.cmake

```diff
@@ -1,24 +1,24 @@
-set(GOPATH "${CMAKE_CURRENT_BINARY_DIR}/go")
-file(MAKE_DIRECTORY ${GOPATH})
+# set(GOPATH "${CMAKE_CURRENT_BINARY_DIR}/go")
+# file(MAKE_DIRECTORY ${GOPATH})
```

Comment:
Why is this commented out?

Reply:
Instead of blaze being a normal go project in the src folder of our GOPATH, it's all the way out here in trueblocks-core/src/go-apps. This goes against "go best practices" about project structure. In response I made modules of the go projects and now we don't need to set a GOPATH.

Reply:
Can you explain what you did to 'make a module'? I don't see a bunch of other code other than that .mod file.

Reply:
@tjayrush I didn't do much to make a module, just ran `go mod init` and did a little sprucing up. [See here about modules](https://blog.golang.org/using-go-modules).